teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Teacher < Person
+  attr_reader :age, :name, :specialization
+
+  def initialize(age, specialization, name = 'Unknown')
+    super(age, name)
```

Comment:
- Hey, let's also add the parent_permission in the parameter as they share this same attribute, the student and teacher from a person.